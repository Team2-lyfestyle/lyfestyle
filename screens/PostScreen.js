import React, { Component } from 'react';
import { Text, StyleSheet, View } from 'react-native';
import { Ionicons } from '@expo/vector-icons';

export default class PostScreen extends Component {
  render() {
    return (
      <View style={styles.container}>
        <View style={styles.header}>
         <Ionicons name="ios-arrow-round-back" size={30} color='#FFF'/>
          <Text style={styles.headerTitle}> POST </Text>
        </View>
      </View>
      
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#EFEdF4'
  },
  header: {
    flexDirection: 'row',
    paddingTop: 40,
    paddingBottom: 8,
    paddingHorizontal: 20,
    backgroundColor: '#204051',
    justifyContent: 'space-between',
    borderBottomWidth: 1,
    borderBottomColor: '#EbECf4',
    shadowColor: '#454D65',
    shadowOffset: { height: 5 },
    shadowRadius: 15,
    shadowOpacity: 0.2,
    zIndex: 10
  },
  headerTitle: {
    fontSize: 15,
    fontWeight: '500',
    color: "#FFF"
  }
});
