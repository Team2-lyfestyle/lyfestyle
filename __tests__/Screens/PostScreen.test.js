import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import PostScreen from '../../screens/PostScreen';

import renderer from 'react-test-renderer';

describe('<PostScreen />', () => {
  it('Should render the <PostScreen /> component', () => {
    const wrapper = shallow(<PostScreen />);
    expect(wrapper.find(PostScreen)).toMatchSnapshot();
  });
});


// test('renders correctly', () => {
//   const tree = renderer.create(<PostScreen />).toJSON();
//   expect(tree).toMatchSnapshot();
// });
