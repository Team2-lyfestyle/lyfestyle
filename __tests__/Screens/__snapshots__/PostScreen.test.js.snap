// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostScreen /> Should render the <PostScreen /> component 1`] = `null`;

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#EFEdF4",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "borderBottomColor": "#CACACA",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingBottom": 8,
        "paddingHorizontal": 20,
        "paddingTop": 40,
        "shadowColor": "#454D65",
        "shadowOffset": Object {
          "height": 5,
        },
        "shadowOpacity": 0.2,
        "shadowRadius": 15,
        "zIndex": 10,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text />
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#000",
            "fontSize": 15,
            "fontWeight": "500",
          }
        }
      >
         POST 
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "margin": 32,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Share your lyfestyle!"
      rejectResponderTermination={true}
      style={
        Object {
          "flexDirection": "row",
        }
      }
      underlineColorAndroid="transparent"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingHorizontal": 20,
      }
    }
  >
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text />
    </View>
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text />
    </View>
  </View>
  <View />
</RCTSafeAreaView>
`;
